src/Octoshift/Services/EnvironmentVariableProvider.cs

```diff
@@ -56,16 +57,27 @@ public EnvironmentVariableProvider(OctoLogger logger)
     public virtual string SmbPassword(bool throwIfNotFound = true) =>
         GetSecret(SMB_PASSWORD, throwIfNotFound);
 
-    private string GetSecret(string secretName, bool throwIfNotFound)
+    public virtual string SkipStatusCheck(bool throwIfNotFound = false) =>
+        GetValue(GEI_SKIP_STATUS_CHECK, throwIfNotFound);
+
+    private string GetValue(string name, bool throwIfNotFound)
     {
-        var secret = Environment.GetEnvironmentVariable(secretName);
+        var value = Environment.GetEnvironmentVariable(name);
 
-        if (string.IsNullOrEmpty(secret))
+#pragma warning disable IDE0046 // Convert to conditional expression
+        if (string.IsNullOrEmpty(value))
         {
             return throwIfNotFound
-                ? throw new OctoshiftCliException($"{secretName} environment variable is not set.")
+                ? throw new OctoshiftCliException($"{name} environment variable is not set.")
                 : null;
         }
+#pragma warning restore IDE0046 // Convert to conditional expression
+
+        return value;
+    }
+    private string GetSecret(string secretName, bool throwIfNotFound)
```

Comment:
nit: newline

Reply:
Addressed in ef5595a23276fcc052a063b906efeb0c1efe2500